File: src/main/java/com/manning/simplysend/mappers/BaseMapper.java
package com.manning.simplysend.mappers;

import java.util.Optional;

import org.springframework.web.util.HtmlUtils;

public abstract class BaseMapper {

    protected static String htmlEscape(String value) {
        return Optional.ofNullable(value).map(HtmlUtils::htmlEscape).orElse("");
    }
}

File: src/main/kotlin/com/manning/simplysend/mappers/BaseMapper.kt
package com.manning.simplysend.mappers

import org.springframework.web.util.HtmlUtils
import java.util.*

abstract class BaseMapper {

    companion object {
        @JvmStatic
        protected fun htmlEscape(value: String?): String {
            return Optional.ofNullable(value).map(HtmlUtils::htmlEscape).orElse("")
        }
    }
}